Sources/SimpleStorageSigner/SimpleStorageSigner+Signing.swift

```diff
@@ -57,6 +57,19 @@ extension SimpleStorageSigner {
     func path(_ url: URL) -> String {
         return !url.path.isEmpty ? url.path.encode(with: .pathAllowed) ?? "/" : "/"
     }
+
+    public func presignedURLV4(httpMethod: HTTPMethod, url: URL, expiration: Expiration, headers: [String: String]) throws -> URL? {
+        let dates = Dates(Date())
+        var updatedHeaders = headers
+        updatedHeaders["Host"] = url.host ?? region.host
```

Comment:
I prefer to use `self` when referencing variables in the class. Can you add that to here `self.region.host`